_posts/2023-03-02-basemap-updates.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Basemaps: Out With the Old, In With the New"
+author:
+  display_name: Zach Beck
+  email: zbeck@utah.gov
+date: 2023-03-02 11:00:00
+categories:
+  - Featured
+tags:
+  - basemaps
+  - cartography
+  - data
+---
+
+![New Terrain basemap]({% link images/new_terrain.png %})
+{: .flex .flex--center}
+
+It’s been about 10 years since the UGRC has updated the look and feel of the basemaps and changes are on the way. The current basemaps were originally authored in ArcMap and have been imported into Pro for the purpose of maintaining the cached raster tiles. Rather than have them limp along in their current form it’s probably best to put them out of their misery and create new basemaps from scratch.
+
+The first basemap on the chopping block to be updated is Terrain. Like the old, the new Terrain basemap will be made up of raster tiles and features a number of improvements including:
+
+- New shaded relief from Jake “255 Shades of Gray” Adams
+- Improved contours
+- 2016 Great Salt Lake level
+- Water related land use agricultural areas
+- ArcGIS Pro symbology
+
+The new Terrain basemap is set to be published in the coming weeks and will overwrite the existing service. If you have the links to the current Terrain basemap saved in a ArcMap or Pro document they will be automatically be replaced by the new service. Since the style of the new Terrain basemap is similar to, but different than the old, overlaid layers symbology may need to be adjusted to the new basemap below.
```

Comment:
```suggestion
The version 2 Terrain base map tiles are set to be published in the coming weeks and will replace the existing tiles. If you have connections to the v1 Terrain base map in a web, ArcMap, or Pro project, the newly styled tiles will start to display after the switch is made. Version 2 will use the same connection url and the version 1 symbology will no longer be accessible. Since the style of the v2 Terrain base map is similar to, but different than v1, overlaid layer symbology may need to be adjusted to visually match the new symbology. 
```